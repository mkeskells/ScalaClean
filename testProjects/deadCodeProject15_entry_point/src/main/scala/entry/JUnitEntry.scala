package junitentry

import org.junit._

class BaseUsed
class BaseUnused

class UsedChild extends Used1
class UnusedChild extends Unused

class Unused extends BaseUnused{
  def used = anotherUsed
  def anotherUsed = ???
  def unused = ???
}

class Used1 extends BaseUsed{
  @Test def used = anotherUsed
  def anotherUsed = ???
  def unused = ???
}
class Used2 extends BaseUsed{
  @Ignore def used = anotherUsed
  def anotherUsed = ???
  def unused = ???
}
class Used3 extends BaseUsed{
  @Before def used = anotherUsed
  def anotherUsed = ???
  def unused = ???
}
class Used4 extends BaseUsed{
  @After def used = anotherUsed
  def anotherUsed = ???
  def unused = ???
}
object Used5 {
  @BeforeClass def used = anotherUsed
  def anotherUsed = ???
  def unused = ???
}
object Used6 {
  @AfterClass def used = anotherUsed
  def anotherUsed = ???
  def unused = ???
}
