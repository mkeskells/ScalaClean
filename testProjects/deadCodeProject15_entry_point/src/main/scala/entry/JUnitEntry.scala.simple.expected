package junitentry

import org.junit._

class BaseUsed
class BaseUnused

class UsedChild extends Used1

class Unused extends BaseUnused{
  def anotherUsed = ???
}

class Used1 extends BaseUsed{
  @Test def used = anotherUsed
  def anotherUsed = ???
}
class Used2 extends BaseUsed{
  @Ignore def used = anotherUsed
  def anotherUsed = ???
}
class Used3 extends BaseUsed{
  @Before def used = anotherUsed
  def anotherUsed = ???
}
class Used4 extends BaseUsed{
  @After def used = anotherUsed
  def anotherUsed = ???
}
object Used5 {
  @BeforeClass def used = anotherUsed
  def anotherUsed = ???
}
object Used6 {
  @AfterClass def used = anotherUsed
  def anotherUsed = ???
}
